requirements.txt

```diff
@@ -1,4 +1,3 @@
-graphviz
 networkx
 matplotlib
```

Comment:
Do we still use matplotlib for anything?

Reply:
Nope